### ISSUE TITLE
Question: why is training set seems to have a lower annotation frame rate?

### ISSUE THREAD

**Issue by jojo23333** [facebookresearch/InterHand2.6M#147]
Hi I was going through the annotation dataset and I can't help notice that:
In the 5fps release version:
frame_idx gap in training set is 18: 01000 -> 01018 -> 01036 ...
in the validation/test set it is 6: 02000 -> 02006 -> 02012 ...

In the 30 fps, the gap is 3 in the training set and 1 (consecutive) in the val/test set

Since I was trying to train a time sequence model on the dataset. I wonder why is this the case? Does this mean the training set annotation is actually lower than 5/30fps in both release?

Thanks so much for your attention!

**Comment by mks0601** [comment]
Hi, you don't have to worry about that. For some captures, the original data is captured in 90 fps, so there could be some inconsistency in the filename, but all the data are properly downsampled to 5 fps or 30 fps

**Comment by jojo23333** [comment]
Thanks so much for the reply! One last question, I also notice that some times there will be missing frames (as you metioned in another issue post). I would like to know that the frame is missing since I'm sending a sequence of frames in the same fps for training. Does it make sense to fix a gap number of 18 for training set and 6 for test set then just detect when consecutive frame id jump over 18/6? (Or it could be that different sequence in training set can have different gap)

**Comment by mks0601** [comment]
I remember those should have 18 interval, while others should have 6 interval. Could you check?
train/Capture0~train/Capture8, train/Capture19, val/Capture0, test/Capture0, test/Capture1
